crates/client/CHANGELOG.md

```diff
@@ -12,12 +12,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
```

Comment:
Had to make the changelog version format compatible with the other crates for release-automation